spec/pg/connection_spec.rb

```diff
@@ -1877,20 +1877,6 @@
 				expect { @conn.set_client_encoding( nil ) }.to raise_error(TypeError)
 			end
 
-			it "can use an encoding with high index for client encoding" do
-				# Allocate a lot of encoding indices, so that MRI's ENCODING_INLINE_MAX is exceeded
-				unless Encoding.name_list.include?("pgtest-0")
-					256.times do |eidx|
-						Encoding::UTF_8.replicate("pgtest-#{eidx}")
-					end
-				end
-
-				# Now allocate the JOHAB encoding with an unusual high index
-				@conn.set_client_encoding "JOHAB"
-				val = @conn.exec("SELECT chr(x'3391'::int)").values[0][0]
-				expect( val.encoding.name ).to eq( "JOHAB" )
-			end
```

Comment:
Note: this test no longer makes sense/can no longer test anything useful since pg does not create encodings dynamically anymore with this PR.